lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -253,6 +258,28 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $security_settings) = @_;
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @cc
+      = map { my ($name, $config) = %$_; ($security_cb->{$name}, $config); } @$security_settings;
```

Comment:
Oh yeah, the "Security Scheme Object"-implementation is referenced via $security_cb->{$name} and the $config is actually the "Security Requirement Object".
This discovery doesn't change the fact that the variable should be named to mean what it is, in this case:
$securityRequirementObject
Had it been so, we wouldn't have had this discussion :)